CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # New Relic Ruby Agent Release Notes #
 
+  ## v8.7.0
+
+  * **Improved async support and Thread instrumentation**
+
+    Previously, the agent was not able to record events and metrics inside Threads created inside of an already running transaction. This release includes 2 new configuration options to support multithreaded applications to automatically instrument threads. To support applications that only want some threads instrumented by New Relic, a new class is available `NewRelic::TracedThread` that will create a thread that includes New Relic instrumentation, see our [API documentation](https://rubydoc.info/github/newrelic/newrelic-ruby-agent/NewRelic) for more details. 
+
+    New configuration options included in this release:
+    | Configuration name | Default | Behavior |
+    | ----------- | ----------- |----------- |
+    | `instrumentation.thread`  | `auto` (enabled) | Allows the agent to correctly nest spans inside of an asyncronous transaction   |
+    | `instrumentation.thread.tracing` | `false` (disabled)   |  Automatically add tracing to all Threads created in the application. This may be enabled by default in a future release. |
+
+
```

Comment:
I think it would be nice to thank the customer who shared his gist with us in this entry as long as Erik thinks it's ok.